src/VirtoCommerce.Platform.Security/Migrations/20210120144559_Initial.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace VirtoCommerce.Platform.Security.Migrations
+{
+    public partial class Initial : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AlterColumn<string>(
```

Comment:
Why did you add this column?

Reply:
This migration is autogenerated from `SecurityDbContextModelSnapshot`

Reply:
@tatarincev approve or decline changes, please. 🔥